src/app/FeedbackResults.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import styled from 'styled-components';
+
+// import { NavLink } from 'react-router-dom';
+// import { library } from '@fortawesome/fontawesome-svg-core';
+// import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+// import { faLightbulb } from '@fortawesome/free-solid-svg-icons';
+import { FeedbackQuestion } from './FeedbackResponse';
+// library.add(faLightbulb);
+
+const FeedbackResponses = styled.section`
+  display: grid;
+  padding: 10px;
+  overflow: scroll;
+`;
+
+export default function FeedbackResultsPage({ responses, questions, counter }) {
+  return (
+    <FeedbackResponses>
+      {questions.map(question => (
+        <FeedbackQuestion
+          key={question.id}
+          questionNumber={question.question_id + 1}
+          question={question.question_text_en}
+          result={(responses[question.question_id] / counter).toFixed(2)}
```

Comment:
for me it is okay. but i am not sure about handling props over to next component plus changing them between the components. a little difficult to write. just ask if you want to talk about that 😆

Reply:
I fixed it like this:

In `FeedbackResults.js`
```javascript
<FeedbackQuestion
          key={question.id}
          questionNumber={question.question_id + 1}
          question={question.question_text_en}
          result={responses}
          index={question.question_id}
          counter={counter}
        />
```

And in `FeedBackResponse.js`

```javascript
export function FeedbackQuestion({
  questionNumber,
  question,
  index,
  result,
  counter,
}) {
  return (
    <FeedbackContainer>
      <QuestionTitle>
        Q{questionNumber}. {question}
      </QuestionTitle>
      <Result>
        {counter !== 0 ? (result[index] / counter).toFixed(2) : 'No Result'}
      </Result>
    </FeedbackContainer>
  );
}
```